observations/obs_converters/MIDAS/MIDAS_to_obs.rst

```diff
@@ -0,0 +1,118 @@
+PROGRAM ``MIDAS_to_obs``
+========================
+
+Overview
+--------
+
+MIDAS netCDF file to DART observation converter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Alex Chartier (University of Bath, UK) is the point-of-contact for this effort.
```

Comment:
I have no idea where he is now ... can't imagine this is useful.  Should we put in an 'out-of-date' disclaimer? An offere to work with someone to revive the effort should they show interest?

Reply:
done.